### DISCUSSION TITLE
Not have an argocd-rollout ui like argocd?

### DISCUSSION THREAD

**Discussion by mod196** [argoproj/argo-rollouts#1393] [Q&A] [answered]
Why doesn't Argocd-Rollout have a UI global administration panel like ArgoCD?

**Comment by mod196** [comment]
i need run argocd-rollout ui localhost? why argocd-rollout don't have an ui with nodeport or svc?

**Comment by jessesuen** [comment, marked as answer]
@Ryu-code we provide a dashboard version https://github.com/argoproj/argo-rollouts/blob/master/manifests/dashboard-install.yaml. Official manifests will be in next v1.1 release, but you can try them out earlier.

**Reply by mod196** [reply]
Yeah!!!
